src/plugins/conditionals/README.md

```diff
@@ -103,20 +103,19 @@ sudo kdb mount main.ini /examples/conditionals ni
 sudo kdb mount sub.ini /examples/conditionals/sub ini
 
 # mount conditionals as global plugin
-sudo kdb global-mount conditionals
+sudo kdb global-mount conditionals || $(exit 0)
```

Comment:
> Why did you only pick 4 out of the 5 commits?

Since I work on macOS and the simpleINI plugin does not work on this OS, pulling in [c0b960ee](https://github.com/ElektraInitiative/libelektra/pull/1301/commits/c0b960ee6f4a92c641a0705127ee22cae79630d4) would only have made things more complicated for me.